packages/twenty-front/src/modules/object-record/record-field/ui/meta-types/input/components/MorphRelationOneToManyFieldInput.tsx

```diff
@@ -40,6 +40,8 @@ export const MorphRelationOneToManyFieldInput = () => {
           ? 'search-bar-on-top'
           : 'search-bar-on-bottom'
       }
+      showObjectName={true}
```

Comment:
For multiple record picker, we can infer this from multipleRecordPickerSearchableObjectMetadataItemsComponentState containing more than 1 element